test/controllers/admin/maintenance_tasks_controller_test.rb

```diff
@@ -25,105 +25,87 @@ class Admin::MaintenanceTasksControllerTest < ActionController::TestCase
       assert_redirected_to teams_path
     end
 
-    test 'get all maintenance tasks' do
+    test 'get all maintenance tasks without ldap' do
       login_as(:admin)
 
       get :index
 
       maintenance_tasks = assigns(:maintenance_tasks)
 
-      assert_equal 1, maintenance_tasks.size
-      assert_equal true, maintenance_tasks.any? { |t| t.class == MaintenanceTasks::RootAsAdmin }
+      assert_equal 0, maintenance_tasks.size
     end
 
-  end
-
-  context '#prepare' do
-
-    test 'prepare cannot be accessed by non-root' do
-      login_as(:bob)
+    test 'get all maintenance tasks with ldap enabled' do
+      MaintenanceTasks::RemovedLdapUsers.any_instance.expects(:enabled?).returns(true).at_least_once
```

Comment:
There is already a method to enable ldap in tests
https://github.com/LBruegger/cryptopus/blob/bug-52743-remove-set-root-as-admin/test/test_helper.rb